pkg/manager/impl/project_manager.go

```diff
@@ -119,6 +120,10 @@ func (m *ProjectManager) UpdateProject(ctx context.Context, projectUpdate admin.
 
 	// Transform the provided project into a model and apply to the DB.
 	projectUpdateModel := transformers.CreateProjectModel(&projectUpdate)
+	err = mergo.Merge(&projectUpdateModel, oldProject)
```

Comment:
why is this necessary? shouldn't the gorm update call already handle merging?

Reply:
you're right. updated it, thanks